app/models/ckeditor/asset.rb

```diff
@@ -1,6 +1,11 @@
+# class Ckeditor::Asset < ActiveRecord::Base
```

Comment:
We can delete the commented code since we have it in version control and keep it cleaner :)